hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
It all depends on where the code gets the pixel width or probe width from. Currently I have been trying to use to x and y-scales from the metadata, and then the emphasis could be on making sure everything is calibrated. However, this causes the problem with the warning on eds_tem.py line 582 because there is a default step size of 1. Therefore it begs the questions of whether or not we add a pixel width option to the metadata or find a way around for line scans. I use line scans for my standards samples and it's pretty safe to assume that the probe area is symmetric in the x and y directions.